CONTRIBUTING.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to PenText
+
+Thanks for taking the time to contribute!
+
+## Pull Requests
+
+PenText uses the [GitHub flow](https://guides.github.com/introduction/flow/) as
+main versioning workflow. In short,
+
+1. Fork the PenText repository
+2. Create a new branch for each feature, fix, or improvement
+3. Send a pull request from each feature branch to the **master** branch
+
+Please create pull requests that are as atomic as possible, and only implement
+one feature, fix, or improvement at a time. That makes it easier to review and
+merge the changes.
+
+## Style Guide
```

Comment:
what are your thoughts on adding pre-commit or pre-push hooks to validate / auto-fix these things?

Reply:
Locally I use pre-commit and commit-msg hooks. I'm not sure if pre-push hooks can be implemented easily on GitHub on a repository. By ensuring that we validate PRs, we can kinda play for 'hook' ;)